data/datasource.go

```diff
@@ -15,7 +15,12 @@ import (
 	"strings"
 	"time"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+
 	"github.com/blang/vfs"
+	gaws "github.com/hairyhenderson/gomplate/aws"
```

Comment:
Obviously, not going to re-use code in your 'aws' package if we decide on the datasource path - just borrowing for now while playing with both